src/main/java/lotto/LottoGame.java

```diff
@@ -0,0 +1,22 @@
+package lotto;
+
+import java.util.List;
+
+public class LottoGame {
+    public static Rank match(List<LottoNumber> userLotto,
+                            List<LottoNumber> winningLotto,
+                            LottoNumber bonusNumber) {
```

Comment:
개인적인 관점으로는 매개 변수가 다소 복잡해 보이기도 하네요 🤔 

```suggestion
    public static Rank match(matchCount, bonusMatch) {
```

이렇게 된다면, 메소드에 대한 책임도 분명해 질 것 같아요.
특히나, 테스트를 하기에 용이해질것 같아요! 🤔